pages/index.js

```diff
@@ -0,0 +1,25 @@
+import Blog from "../components/section/Blog";
+import Footer from "../components/section/Footer";
+import Project from "../components/section/Project";
+import Header from "../components/layout/Header";
```

Comment:
About the folder structure folders containing multiple components should be named in the plural so these components should be imported from `sections` not `section`